### PR TITLE
restart hummer v3

### PR DESCRIPTION
**Pull request by max-lobur**

### CODE REVIEW

**Comment by max-lobur** [comment]
[ch15678]

File: templates/logrotate.j2 [hunk 1]
```diff
@@ -1,6 +1,5 @@
 /var/log/td-agent/td-agent.log {
-  size 100M
```
Comment (akhon) [inline templates/logrotate.j2]: we had problems with large log files, which we may get during the day.
let's leave this option, adjust if needed

Comment (max-lobur) [reply templates/logrotate.j2]: without this option it will truncate every time, not waiting until log size is 100

Comment (akhon) [reply templates/logrotate.j2]: still not solving possible issue.
size + hourly rotation?

**Pull request merged**